screed/fadbm.py

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
 
-# Copyright (c) 2008-2010, Michigan State University
+# Copyright (C) 2008-2010, 2014-2015 Michigan State University
```

Comment:
Is this the correct way of specifying copyright? I checked when the file was changed (no changes from 2011 to 2013), but I think this should be `2008-2015` anyway...